src/content/docs/infrastructure/prometheus-integrations/integrations-list/install-apache-flink.mdx

```diff
@@ -76,6 +78,10 @@ While the infrastructure agent should send logs to Flink dashboards, you may nee
         logtype: flink-logs
 ```
 
+## Get Apache Flink metrics as a dashboard
+
+Once you've installed the [Apache Flink quickstart](https://newrelic.com/instant-observability/redpanda), you can see your critical Apache Flink data in New Relic. To find your dashboard in New Relic: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards**, then select **Apache Flink**. 
+
```

Comment:
Can you add the below text too..

Run the following NRQL query: `FROM Metric SELECT sum(flink_jobmanager_job_totalNumberOfCheckpoints) AS 'Total Number of Checkpoints'`